src/Repository/ProgrammeRepository.php

```diff
@@ -99,8 +110,9 @@ public function getPaginatedFilteredSorted(
             $direction = 'ASC';
         }
 
-        if ('' != $sort) {
-            $query = $query->orderBy("p.$sort", $direction);
```

Comment:
is there a validation of the value in $sort done anywhere? It should be validated that is a valid field that can be sorted upon. Unless there is such a validation, this code is dangerous.